docs/NOW-spec.md

```diff
@@ -235,10 +235,10 @@ packet-beta
   48-63: "msgFlags"
   64-79: "versionMajor"
   80-95: "versionMinor"
-  96-127: "systemCapset"
-  128-159: "sessionCapset"
-  160-191: "execCapset"
-  192-223: "heartbeatInterval"
+  96-111: "systemCapset"
+  112-127: "sessionCapset"
+  128-143: "execCapset"
+  144-175: "heartbeatInterval"
```

Comment:
issue: I’m pretty sure this could be considered as a breaking change since the on wire representation is not compatible anymore with version 1.0. If we use the decoder for v1.0, it will breaks.

suggestion: Either,
- do not make this modification, or
- perform a major bump to 2.0, and add checks for fast-failing when using incompatible versions.

Reply:
Oh, so it’s just about updating the doc to match the current implementation? Not a problem then.

Side note: it would help to have a separate PR dedicated to this change so it’s easier to understand what is happening.